lib/betamocks/configuration.rb

```diff
@@ -51,7 +51,7 @@ def service_by_host_port(env)
     end
 
     def matches_path(endpoint, method, path)
-      /#{endpoint[:path].gsub('/', '\/').gsub('*', '.*')}/ =~ path && endpoint[:method] == method
+      /\A#{endpoint[:path].gsub('/', '\/').gsub('*', '[^\/]*')}\z/ =~ path && endpoint[:method] == method
```

Comment:
Took me longer than I care to admit to understand what was going on with `[^\/]`.  

This relies on an additional `/` char.  For example, it would not be able to disambiguate the following routes:
- `/cats/1`
- `/cats/toys`

I'll LGTM for now, but I suspect we may have more matching issues in the future.